ducktape/command_line/main.py

```diff
@@ -56,6 +57,7 @@ def parse_args():
                              "This is primarily useful for test developers who want to interact with running " +
                              "services after a test has run. ")
     parser.add_argument("--version", action="store_true", help="display version")
+    parser.add_argument("--inject", action="store", help="Inject these arguments into the specified test. Use JSON to format.")
```

Comment:
Yeah, I considered `--args` or `--arguments`. `--args` has the obvious benefit of brevity. `--parameters` has the benefit of consistency. I'd be happy with either one.